packages/clerk-js/src/ui/portal/index.tsx

```diff
@@ -11,28 +13,29 @@ type PortalProps<CtxType extends AvailableComponentCtx, PropsType = Omit<CtxType
   node: HTMLDivElement;
   component: React.FunctionComponent<PropsType> | React.ComponentClass<PropsType, any>;
   // Aligning this with props attributes of ComponentControls
-  props?: PropsType & { path?: string; routing?: string };
+  props: PropsType & RoutingOptions;
```

Comment:
Why did we decide to add this to the `portal` level?

Reply:
It's the same behaviour but I made the types stricter everywhere.

Reply:
Pretty sure it's not a public type.